src/controller/CHIPDeviceController.cpp

```diff
@@ -42,18 +42,17 @@
 #include <core/CHIPCore.h>
 #include <core/CHIPEncoding.h>
 #include <core/CHIPSafeCasts.h>
+#include <errno.h>
```

Comment:
the previous order was deliberate, plase put these system headers last

Reply:
This is caused by formatter issues on my side, fixed now.

Reply:
cool, np.  ./scripts/helpers/restyle-diff.sh runs the CI reformatter.  as long as the groups of headers are separated by a linefeed, the order of the groups is preserved